lib/companies/pending_changes.ex

```diff
@@ -9,32 +9,15 @@ defmodule Companies.PendingChanges do
   alias Companies.Schema.{Company, Job, Industry, PendingChange}
 
   def all do
-    query =
-      from p in PendingChange,
-        select: p,
-        where: p.approved == false
+    query = from p in PendingChange, where: p.approved == false
```

Comment:
What was the thinking behind this change?

Reply:
I found the select redundant, so I removed it.